data/phi1348/abo011/__cts__.xml

```diff
@@ -1,11 +1,12 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1348.abo011" xml:lang="lat" groupUrn="urn:cts:latinLit:phi1348">
             <ti:title xml:lang="lat">Divus Julius</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi1348.abo011.perseus-lat2" workUrn="urn:cts:latinLit:phi1348.abo011">
-              <ti:label xml:lang="eng">Divus Julius, De Vita Caesarum Libri VIII</ti:label>
-              <ti:description xml:lang="eng">Suetonius, ca. 69-ca. 122, creator; Ihm, Max, 1863-1909, editor</ti:description>
+              <ti:label xml:lang="lat">Divus Julius</ti:label>
+              <ti:description xml:lang="mul">Suetonius. De Vita Caesarum Libri VIII. Ihm, Max, editor; Leipzig: Teubner, 1908.</ti:description>
             </ti:edition>
           <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi1348.abo011.perseus-eng2" workUrn="urn:cts:latinLit:phi1348.abo011">
-              <ti:label xml:lang="eng">Divus Julius, The Lives of the twelve Caesars</ti:label>
-              <ti:description xml:lang="eng">Suetonius, ca. 69-ca. 122, creator; Thomson, Alexander, M.D, translator; Reed, J.E. (J. Eugene), editor</ti:description>
+              <ti:label xml:lang="eng">Julius Caesar</ti:label>
+            <ti:description xml:lang="eng">Suetonius. Suetonius: The Lives of the Twelve Caesars. Thomson, Alexander, M.D, translator; Reed, J.E. (J. Eugene), editor.
+            Philadelphia:  Gebbie &amp; Commpany, 1889.</ti:description>
```

Comment:
I think we determined that `&amp;` shows up in the Scaife TOC, yes? I generally change these to "and" to avoid that.